src/home/components/HomeCard.js

```diff
@@ -16,20 +16,85 @@
  */
 import React from 'react';
 
-import _ from 'lodash/fp';
+import {
+  Box,
+  Card,
+  CardActions,
+  Divider,
+  Link,
+  Paper,
+  Stack,
+  Typography,
+} from '@mui/material';
 
-import { Card } from '@mui/material';
+import CardActionRouterLink from 'common/components/CardActionRouterLink';
+import RouterButton from 'common/components/RouterButton';
+import RouterLink from 'common/components/RouterLink';
 
-const HomeCard = props => (
+const HomeCard = ({
+  id,
+  title,
+  action,
+  image,
+  children,
+  isDefault,
+  useImg = true,
```

Comment:
suggestion: rename this to `useImage` (or perhaps `hasImage`).

Reply:
I think you should be able to check if the `image` prop was passed and check you need to render an image